src/main/steps/c100-rebuild/screening-questions/permission/content.test.ts

```diff
@@ -0,0 +1,94 @@
+import languageAssertions from './../../../../../test/unit/utils/languageAssertions';
+import { FormContent, FormFields, FormOptions, LanguageLookup } from './../../../../app/form/Form';
+import { CommonContent, generatePageContent } from './../../../common/common.content';
+import { generateContent } from './content';
+
+jest.mock('./../../../../app/form/validation');
+
+const en = {
+  title: 'Is there any reason why you would need permission from the court to make this application?',
+  paragraphsOne: [
+    'In some cases, the court will need to give permission before the application can be made.',
+    'You will not need permission if you are the:',
+  ],
+  bulletPoints: ['parent', 'guardian', 'special guardian'],
+  paragraphsTwo: [
+    'You may, or may not need permission from the court.',
+    'For example, you may need permission if you are in the children’s lives, but not their parent or guardian.',
+    'You will need the court’s permission if you are one of the children, and the application is about you.',
+    'If you are not sure if you need permission, <a  target="_blank" href="https://www.gov.uk/government/publications/family-court-applications-that-involve-children-cb1">see Section C of the Form CB1 guidance on GOV.UK</a>',
+  ],
+  one: 'Yes',
+  two: 'No',
+  errors: {
+    sq_courtPermissionRequired: {
+      required:
+        'Select yes if there is any reason why you would need permission from the court to make this application',
+    },
+  },
+};
+
+const cy = {
+  title: 'Is there any reason why you would need permission from the court to make this application?',
```

Comment:
Can you please add - welsh at the end of each attribute .  
For e.g. Is there any reason why you would need permission from the court to make this application? - welsh

Reply:
done